src/components/Materialize/MSelect.vue

```diff
@@ -6,6 +6,11 @@
   export default {
     id: 'm-select',
     twoWay: true,
+    update (value) {
```

Comment:
This is a directive. It shouldn't be a component with an empty template nor in a .vue file